aten/src/ATen/native/PointwiseOps.cpp

```diff
@@ -69,6 +69,18 @@ Tensor& addcdiv_out(
     const Tensor& tensor1,
     const Tensor& tensor2,
     Scalar value) {
+  if (isIntegralType(tensor1.scalar_type(), /*includeBool=*/ true)
+      && isIntegralType(tensor2.scalar_type(), /*includeBool=*/ true)) {
+    TORCH_WARN_ONCE(
+      "Integer division with addcdiv is deprecated. ",
+      "In PyTorch 1.6 it will be disabled, and in PyTorch 1.7 addcdiv will a ",
```

Comment:
I think we should just be more aggressive with addcdiv and change the behavior in 1.6 because I think addcdiv is not commonly used.

Reply:
Made this statement vaguer.